src/components/scenes/SweepPrivateKeyCalculateFeeScene.tsx

```diff
@@ -201,7 +201,7 @@ const SweepPrivateKeyCalculateFeeComponent = (props: Props) => {
         <FlatList
           automaticallyAdjustContentInsets={false}
           data={sweepPrivateKeyList}
-          extraData={transactionState}
+          extraData={exchangeRates}
```

Comment:
Include both exchangeRates and transactionState in the extraData as memoized state